sql/moz-fx-data-shared-prod/contextual_services/quicksuggest_impression_live/view.sql

```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.contextual_services.quicksuggest_impression_live`
+AS
+SELECT
+  * EXCEPT (search_query, matched_keywords) REPLACE(mozfun.norm.metadata(metadata) AS metadata)
+FROM
+  `moz-fx-data-shared-prod.contextual_services_live.quicksuggest_impression_v1`
```

Comment:
The logic here is identical to the non-live view that's already in this directory. It just references the live table rather than the stable one.